tokio-stream/src/lib.rs

```diff
@@ -81,7 +81,20 @@ pub mod wrappers;
 
 mod stream_ext;
 pub use stream_ext::{collect::FromStream, StreamExt};
+/// Adapters for [`Stream`]s created by methods in [`StreamExt`].
+pub mod adapters {
+    pub use crate::stream_ext::{
+        AllFuture, AnyFuture, Chain, Filter, FilterMap, FoldFuture, Fuse, Map, MapWhile, Merge,
+        Next, Peekable, Skip, SkipWhile, Take, TakeWhile, Then, TryNext,
+    };
```

Comment:
Actually, can we drop `Next` from the list? I'd like to only export the ones that implement `Stream`, not the ones that implement `Future`.

Reply:
Sure!

Reply:
Done!

Reply:
It looks like there are a few more futures. Can you remove those too? You can view the generated docs with your change [here](https://deploy-preview-6658--tokio-rs.netlify.app/tokio_stream/adapters/index.html). 

Also, we should probably move `Elapsed` to the root. It's an error type.

Reply:
The preview is very cool!
Done!

Reply:
I removed `Elapsed` from the `doc(hidden)` and `deprecated` combo as well, since it is the only way to access it.